ykllvmwrap/src/symbolizer.rs

```diff
@@ -68,8 +68,8 @@ mod tests {
         let sym = s.find_code_sym(f_addr).unwrap();
         //let sym_str = sym.to_str().unwrap();
         // The symbol will be suffixed with an identifier for the tests module, e.g.:
-        // ykllvm::symbolizer::tests::symbolize_me_mangled::hc7a76ddceae6f9c4
-        assert!(sym.starts_with("ykllvm::symbolizer::tests::symbolize_me_mangled::"));
+        // ykllvmwrap::symbolizer::tests::symbolize_me_mangled::hc7a76ddceae6f9c4
```

Comment:
I'm not sure I understand the comment.

Reply:
What I'm trying to convry is that the `hc7a76ddceae6f9c4` junk is an auto-generated module name out of our control.

Reply:
Ah, I get it. Ignore me -- I was being an idiot!

Reply:
I'll reword it a little nonetheless.